### PR TITLE
Terms of Service Feature Integrated

### PR DESCRIPTION
**Pull request by Vaibhav-22-dm**
Used the django-tos package to implement Terms of Service Feature. 
Added custom template for tos_check.html page. 
Fixed the Unicode error using Monkey Patching.

### CODE REVIEW

File: vmb/templates/tos/tos_check.html [hunk 1]
```diff
@@ -0,0 +1,99 @@
+{% load i18n %}
```
Comment (rtnpro) [inline vmb/templates/tos/tos_check.html]: extend this from `vmb/templates/base_minimal.html`

File: config/settings/local.py [hunk 2]
```diff
@@ -21,9 +21,17 @@
     "default": {
         "BACKEND": "django.core.cache.backends.locmem.LocMemCache",
         "LOCATION": "",
+    },
```
Comment (rtnpro) [inline config/settings/local.py]: We should add these tos settings in `settings/base.py` and use `redis` as a cache for it. Then, we don't need to override the settings `local` and `production` as we have redis in both the places.

File: config/settings/local.py [hunk 3]
```diff
@@ -35,6 +43,7 @@
 # ------------------------------------------------------------------------------
 # https://django-debug-toolbar.readthedocs.io/en/latest/installation.html#prerequisites
 INSTALLED_APPS += ["debug_toolbar"]  # noqa F405
+# INSTALLED_APPS += ["tos"] # adding the tos app of django-tos package
```
Comment (rtnpro) [inline config/settings/local.py]: This line is not needed

File: vmb/matrimony/models/profiles.py [hunk 4]
```diff
@@ -1698,3 +1698,10 @@ class MatrimonyProfileStats(BaseModel):
 
     class Meta:
         db_table = "matrimony_profile_stats"
+
+from tos.models import UserAgreement
```
Comment (rtnpro) [inline vmb/matrimony/models/profiles.py]: A better way will be to create a new app called `tos_custom` to do this monkey patching

**Review by rtnpro — CHANGES_REQUESTED** [review]
Also remove the files under `media/photologue/*` from this pull request. They are not related and seem to be related to your local development setup.

My suggestion will be to selectively add files for a git commit using `git add path/to/file(s)` rather than doing `git add .`

File: vmb/matrimony/models/profiles.py (hunk 4 above)
Comment (Vaibhav-22-dm) [reply vmb/matrimony/models/profiles.py]: Addressed and Resolved.

**Comment by Vaibhav-22-dm** [comment]
Requested changes have been done.

**Review by rtnpro — APPROVED** [review]

**Pull request merged**